File: mcp-server/src/__fixtures__/typescript/good.ts
function add(a: number, b: number): number {
  return a + b;
}

const MAX_RETRIES = 3;

async function fetchData(url: string): Promise<unknown> {
  try {
    const response = await fetch(url);
    return response.json();
  } catch (err) {
    console.error("fetch failed", err);
    throw err;
  }
}

export { add, MAX_RETRIES, fetchData };

File: mcp-server/src/__fixtures__/python/good.py
def add(a: int, b: int) -> int:
    return a + b

MAX_RETRIES = 3

def fetch_data(url: str) -> dict:
    try:
        import requests
        response = requests.get(url)
        return response.json()
    except Exception as err:
        print("fetch failed", err)
        raise
